libs/core/src/lib/bar/button-bar/button-bar.component.ts

```diff
@@ -23,8 +23,8 @@ import { ContentDensityService } from '@fundamental-ngx/core/utils';
               [compact]="compact"
               [fdType]="fdType"
               [label]="label"
-              [attr.title]="title"
-              [attr.aria-label]="ariaLabel"
+              [attr.title]="ariaLabel"
+              [ariaLabel]="ariaLabel"
```

Comment:
I'm confused. Why did you change [attr.aria-label] to [ariaLabel]?

Reply:
this change is overridden when ariaLabel is passed or label is passed as a parameter.